docs/htdocs/info/docs/api/variation/variation_tutorial.html

```diff
@@ -983,21 +1000,27 @@ <h2 id="alleles">Alleles and frequencies</h2>
   <h3>Retrieve external data (e.g. 1000 Genomes phase 3)</h3>
   <div class="message-pad">
     <p>To retrieve external data e.g. from the 1000 Genomes phase 3 in human, remember to set the flag <b>"use_vcf"</b> to <span style="color:#F00;font-weight:bold">1</span> before making a call to retrieve alleles, genotypes or LD data. See an example in the following code:</p>
-    <pre class="code sh_perl">my $variation_adaptor = $registry-&gt;get_adaptor("human", "variation", "variation");
+    <pre class="code sh_perl">
```

Comment:
This needs updating to reflect that it's not just 1KG with data outside the db. There are more results if you switch VCF usage on for this example too.

How about 'To retrieve external data e.g. from the 1000 Genomes phase 3 in human, remember to set the flag ' =>
Additional frequency data is available via our VCF API extensions. Add the line
$variation_adaptor->db->use_vcf(1)
in the aboue example to retrieve frequencies from the WSI  Mouse Genomes Project